src/NUnitFramework/nunitlite/nunitlite.csproj

```
@@ -6,13 +6,16 @@
     <Nullable>disable</Nullable>
   </PropertyGroup>
 
-  <ItemGroup>
-    <ProjectReference Include="..\framework\nunit.framework.csproj" />
-  </ItemGroup>
+  <PropertyGroup Condition="'$(TargetFramework)' == 'net462'">
+    <Title>NUnitLite Runner (.NET Framework 4.6.2)</Title>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(TargetFramework)' == 'net6.0'">
+    <Title>NUnitLite Runner (.NET 6.0)</Title>
+  </PropertyGroup>
+  
```

Maybe we can use the previously set `AssemblyConfiguration` in order not to duplicate things:

```suggestion
  <PropertyGroup>
    <Title>NUnitLite Runner $(AssemblyConfiguration)</Title>
  </PropertyGroup>
```